packages/execute-driver/test/e2e/plugin-e2e-specs.js

```diff
@@ -0,0 +1,154 @@
+import path from 'path';
+const APPIUM_HOME = path.resolve(__dirname, '..', '..', '..', 'local_appium_home');
+// need to assign this before appium is imported via e2eSetup
+process.env.APPIUM_HOME = APPIUM_HOME;
+
+import { e2eSetup } from '@appium/base-plugin/build/test/helpers';
+import { remote as wdio } from 'webdriverio';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { W3C_ELEMENT_KEY, MJSONWP_ELEMENT_KEY } from '../../lib/execute-child';
+
+chai.use(chaiAsPromised);
+const should = chai.should();
+
+const THIS_PLUGIN_DIR = path.resolve(__dirname, '..', '..', '..');
+const TEST_HOST = 'localhost';
+const TEST_PORT = 4723;
+const TEST_FAKE_APP = path.resolve(APPIUM_HOME, '@appium', 'fake-driver',
```

Comment:
due to `__dirname`, `APPIUM_HOME` is already absolute, so `path.join` would work just as well